x-pack/plugins/index_lifecycle_management/public/application/services/policies/types.ts

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export interface SerializedPolicy {
+  name: string;
+  phases: Phases;
+}
+
+export interface Phases {
+  hot?: SerializedHotPhase;
+  warm?: SerializedWarmPhase;
+  cold?: SerializedColdPhase;
+  delete?: SerializedDeletePhase;
+}
+
+export interface PolicyFromES {
+  modified_date: string;
+  name: string;
+  policy: SerializedPolicy;
+  version: number;
+}
+
+export interface SerializedPhase {
+  min_age: string;
+  actions: {
+    [action: string]: any;
+  };
+}
+
+export const serializedPhaseInitialization: SerializedPhase = {
```

Comment:
I would prefer not to mix constants inside a "types.ts" file. Can this be moved to a constant file? Maybe the same as I mentioned above?